src/locales/en-US.ts

```diff
@@ -1,16 +1,18 @@
 import component from './en-US/component';
 import globalHeader from './en-US/globalHeader';
 import menu from './en-US/menu';
+import pages from './en-US/pages';
 import pwa from './en-US/pwa';
+import request from './en-US/request';
 import settingDrawer from './en-US/settingDrawer';
 import settings from './en-US/settings';
-import pages from './en-US/pages';
 
 export default {
   'navBar.lang': 'Languages',
   'layout.user.link.help': 'Help',
   'layout.user.link.privacy': 'Privacy',
   'layout.user.link.terms': 'Terms',
+  'app.copyright.produced': 'Produced by Ant Group Experience Technology Department',
```

Comment:
This was the translation that Google gave me, but I'm not sure if "Ant Group Experience Technology Department" is the correct translation, so just let me know.

Reply:
![image](https://user-images.githubusercontent.com/8186664/104978699-6e010e80-5a3d-11eb-910c-1973435fe8ec.png)


**Ant Financial Experience**